Tools/scripts/extract_features.py

```diff
@@ -62,6 +62,8 @@ def __init__(self, filename, nm="arm-none-eabi-nm", strings="strings"):
             ('HAL_EFI_ENABLED', 'AP_EFI::AP_EFI',),
             ('AP_EFI_{type}_ENABLED', 'AP_EFI_(?P<type>.*)::update',),
 
+            ('AP_EXTENDED_ESC_TELEM_ENABLED', 'AP_Extended_Telem::get_input_duty',),
```

Comment:
I suspect this won't work because the `get_input_duty` function is not used anywhere so the linker will remove it.

Reply:
The APD ESC extended status telemetry messages implementation utilizes these functions. Given its specificity to our requirements, we decided to create two separate pull requests.

The first pull request includes the driver implementation (which is this one), as we believe it could be beneficial to a broader audience. 
The second pull request will contain the AP_Periph interface with the APD ESC, which we plan to submit at a later time.